packages/dds/merge-tree/src/test/mergeTree.insertingWalk.spec.ts

```diff
@@ -38,13 +38,15 @@ const treeFactories: ITestTreeFactory[] = [
         create: () => {
             const initialText = "hello world";
             const mergeTree = new MergeTree();
```

Comment:
@Abe27342 -- the merge-tree test passed after properly merging `mergeTree.insertingWalk.spec.ts`. it would be great if you could check this file. thanks!

Reply:
the file looks correct in most recent commit. I suspect it was a bad merge of the change Tony made here https://github.com/microsoft/FluidFramework/pull/12190/files in `next` and Connor refactoring the function signatures in main. Confirming that's the case on the pipeline branch that had failed tests.

Reply:
yup, it's exactly that. see here https://github.com/microsoft/FluidFramework/blob/a9a544afa60ea3c0718b4da2c50afe22597ae2a2/packages/dds/merge-tree/src/test/mergeTree.insertingWalk.spec.ts#L158